src/DynamoCoreWpf/Controls/DynamoTextBox.cs

```diff
@@ -387,13 +387,6 @@ private void LoadMainDynamoWindow(object sender, RoutedEventArgs e)
 
         public void SetDataContext(object dataContext, bool closeImmediately = false)
         {
-            if (mainDynamoWindow == null)
```

Comment:
You can remove this,  for sure. But please update the statement below to `if (mainDynamoWindow == null || !mainDynamoWindow.IsActive) return;`